plugins/catalog/src/js/repositories/RepositoriesDelete.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import { componentFromStream } from "data-service";
```

Comment:
Hum. I started by having "#PACKAGES". I think there is a silent understand from the past that `packages` are an incubator for things that will be packaged on their on (which is the case for the data layer).

I like the way it is because when we move it to be a package (which will 100% happen) that means the client code does not need change.